languages/rust/exercises/concept/structs/tests/structs.rs

```diff
@@ -0,0 +1,52 @@
+use structs::*;
+
+const NAME: &str = "Ebenezer";
+const AGE: u32 = 89;
+const WEIGHT: f32 = 131.6;
+
+#[test]
+fn test_get_name() {
+    let new_user = User::new(NAME.to_string(), AGE, WEIGHT);
```

Comment:
```suggestion
    let user = User::new(NAME.into(), AGE, WEIGHT);
```

`user` instead of `new_user` usually reads better unless there is a need for disambiguation. `NAME.into()` is just a bit shorter / faster to type than `NAME.to_string()`.